src/main/java/com/uid2/admin/Main.java

```diff
@@ -204,15 +205,16 @@ public void run() {
             try {
                 s3KeyProvider.loadContent();
             } catch (CloudStorageException e) {
-                if (e.getMessage().contains("s3 get error")) {
+                if (e.getMessage().contains("s3 get error") || e.getMessage().contains("The specified key does not exist")) {
                     s3KeyStoreWriter.upload(new HashMap<>(), null);
                     s3KeyProvider.loadContent();
                 } else {
                     throw e;
                 }
             }
             s3KeyManager.generateKeysForOperators(operatorKeyProvider.getAll(), config.getLong("s3_key_activates_in_seconds"), config.getInteger("s3_key_count_per_site"));
-
+            Thread.sleep(8000); // Wait before checking again
```

Comment:
Issue: When admin get started for the first time, some times it takes a few times of running to fully generate the whole s3 encryption keys file